app/utils/index.js

```diff
@@ -1,18 +1,18 @@
 const defaultConfig = {
-  // campaignReference: "ce84ab7f7b737d",
   // campaignName: "https://lviv.com",
```

Comment:
name, and reference
These should be removed in final version of code.

Reply:
@Bilal-S Yes, it is for testing now